code/game/machinery/computer/crew.dm

```diff
@@ -97,128 +96,90 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
-			if (H.z == 0 && (!pos || pos.z != z))
-				continue
-
-			id_card = H.wear_id ? H.wear_id.GetID() : null
-
-			if (id_card)
-				name = id_card.registered_name
-				assignment = id_card.assignment
-				ijob = jobs[id_card.assignment]
-			else
-				name = "Unknown"
-				assignment = ""
-				ijob = 81
-
-			life_status = (!H.stat ? TRUE : FALSE)
 
-			oxydam = round(H.getOxyLoss(),1)
-			toxdam = round(H.getToxLoss(),1)
-			burndam = round(H.getFireLoss(),1)
-			brutedam = round(H.getBruteLoss(),1)
-
-			area = get_area_name(H, TRUE)
-			pos_x = pos.x
-			pos_y = pos.y
-
-			results[++results.len] = list("name" = name, "assignment" = assignment, "ijob" = ijob, "life_status" = life_status, "oxydam" = oxydam, "toxdam" = toxdam, "burndam" = burndam, "brutedam" = brutedam, "area" = area, "pos_x" = pos_x, "pos_y" = pos_y, "can_track" = H.can_track(null))
-
-	for(var/i in GLOB.suit_sensors_list)
-		var/mob/living/carbon/human/H = i
-		// Check if their z-level is correct and if they are wearing a uniform.
-		// Accept H.z==0 as well in case the mob is inside an object.
-		// Also exclude people already listed due to nanite sensors (which will always be at maximum detail)
-		if((H.z == 0 || H.z == z) && (istype(H.w_uniform, /obj/item/clothing/under)) && !(H in GLOB.nanite_sensors_list))
-			uniform = H.w_uniform
-
-			// Are the suit sensors on?
-			if (((uniform.has_sensor > 0) && uniform.sensor_mode))
-				pos = get_turf(H)
-
-				// Special case: If the mob is inside an object confirm the z-level on turf level.
-				if (H.z == 0 && (!pos || pos.z != z))
-					continue
-
-				id_card = H.wear_id ? H.wear_id.GetID() : null
-
-				if (id_card)
-					name = id_card.registered_name
-					assignment = id_card.assignment
-					ijob = jobs[id_card.assignment]
-				else
-					name = "Unknown"
-					assignment = ""
-					ijob = 81
-
-				if (uniform.sensor_mode >= SENSOR_LIVING)
-					life_status = (!H.stat ? TRUE : FALSE)
-				else
-					life_status = null
-
-				if (uniform.sensor_mode >= SENSOR_VITALS)
-					oxydam = round(H.getOxyLoss(),1)
-					toxdam = round(H.getToxLoss(),1)
-					burndam = round(H.getFireLoss(),1)
-					brutedam = round(H.getBruteLoss(),1)
-				else
-					oxydam = null
-					toxdam = null
-					burndam = null
-					brutedam = null
-
-				if (pos && uniform.sensor_mode >= SENSOR_COORDS)
-					area = get_area_name(H, TRUE)
-					pos_x = pos.x
-					pos_y = pos.y
-				else
-					area = null
-					pos_x = null
-					pos_y = null
-
-				results[++results.len] = list("name" = name, "assignment" = assignment, "ijob" = ijob, "life_status" = life_status, "oxydam" = oxydam, "toxdam" = toxdam, "burndam" = burndam, "brutedam" = brutedam, "area" = area, "pos_x" = pos_x, "pos_y" = pos_y, "can_track" = H.can_track(null))
-
-	data_by_z["[z]"] = sortTim(results,/proc/sensor_compare)
+		// Check if z-level is correct
+		// Accept z == 0 as well in case mob is inside an object
+		if (H.z != 0 && H.z != z)
+			continue
```

Comment:
Why not have this and everything else that checks `z != 0` just use `get_turf(H).z`?